src/schema/2.0/schema.test.ts

```diff
@@ -298,7 +303,7 @@ describe("schema/v2.0", () => {
             {
               name: "ABC",
               identityProof: {
-                type: "DNS-TXT",
+                type: IdentityProofType.DNSTxt,
```

Comment:
Why did you change some of these and not those on top? I'll honestly prefer to keep them as "DNS-TXT" in the test to show that the type didnt change after this PR though

Reply:
will see in another MR